sharktank/sharktank/ops/default_impls.py

```diff
@@ -601,6 +601,11 @@ def sharded_cat_unsharded(maybe_sharded):
     return unbox_tensor(maybe_sharded)
 
 
+@sharded_gather.override(Tensor)
+def sharded_gather_unsharded(input):
+    return input
```

Comment:
Does this need a default implementation for unsharded?

If so, I think it should be returning `[input]` so the return type is consistent and can be used interchangable.

Reply:
@sogartar Thoughts?

Reply:
Yes, lets return a list for consistency.